### PR TITLE
Dashboard Page / Widget Layout / Progress Reports Grids

### PR DESCRIPTION
**Pull request by andrewmurraydavid**
Add the TeamNews skeleton/foundation for My Dashboard page

[Monday Task](https://seed-company-squad.monday.com/boards/5989610236/views/132288519/pulses/7125527025)

### CODE REVIEW

File: src/components/Widgets/TeamNews/TeamNewsWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    },
+    {
+      headerName: 'Team News',
+      type: 'string',
+      field: 'project.primaryLocation',
+      valueGetter: (_, row) => row.project.primaryLocation.value?.name.value,
+      renderCell: () => (
+        <Box
+          sx={{
+            height: '100%',
```
Comment (willdch) [inline src/components/Widgets/TeamNews/TeamNewsWidget.tsx:75]: Maybe? 
```css
height : 1
```
i believe this is does the same as `height : '100%'`.  If I am wrong then ignore.

File: src/components/Widgets/TeamNews/TeamNewsWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      ),
+    },
+    {
+      headerName: 'Stories',
+      type: 'string',
+      field: 'project.step',
+      renderCell: () => (
+        <Box
+          sx={{
+            height: '100%',
```
Comment (willdch) [inline src/components/Widgets/TeamNews/TeamNewsWidget.tsx:92]: same here?
```css
height : 1
```

**Review by willdch — APPROVED** [review]
Looks good.  Ignore my comments if they hold no value.

**Review by CarsonF — APPROVED** [review]

**Comment by rdonigian** [comment]
LGTM

**Review by rdonigian — APPROVED** [review]

**Pull request merged**